### DISCUSSION TITLE
Latest Version Clarification

### DISCUSSION THREAD

**Discussion by KeonPaul** [coreybutler/nvm-windows#801] [General] [answered]
I apologize if this question has been addressed previously but, is there a reason why the Releases tab still marks [v1.1.1](https://github.com/coreybutler/nvm-windows/releases/tag/1.1.1) as the latest version when [v1.1.9](https://github.com/coreybutler/nvm-windows/releases/tag/1.1.9)  is available. Especially since using v1.1.1 still utilizes the old npm repository which can cause errors while trying to install node. Using the download now! button on the README links you directly to the Release Tab and new user may not look at the tag section.

**Comment by coreybutler** [comment, marked as answer]
Perhaps there was a Github glitch when this post was made. The latest version is showing up now:

<img width="287" alt="image" src="https://user-images.githubusercontent.com/770982/179400555-1243c126-f81b-4209-832c-54df6d32e3f7.png">
